bluepyentity/utils.py

```diff
@@ -82,3 +85,35 @@ def get_secret(prompt):
         stream.flush()  # issue7208
 
     return passwd
+
+
+def url_get_revision(url: str) -> bool:
```

Comment:
should the type hint return value be `Optional[int]`?

Reply:
yep